pkgs/top-level/perl-packages.nix

```diff
@@ -17596,9 +17600,11 @@ let
       url = mirror://cpan/authors/id/T/TO/TODDR/XML-Parser-2.44.tar.gz;
       sha256 = "05ij0g6bfn27iaggxf8nl5rhlwx6f6p6xmdav6rjcly3x5zd1s8s";
     };
-    patchPhase = if stdenv.isCygwin then ''
+    patchPhase = stdenv.lib.optionalString (stdenv.buildPlatform != stdenv.hostPlatform) ''
+      substituteInPlace Expat/Makefile.PL --replace 'use English;' '#'
```

Comment:
Is this module not used at all in the file? I am surprised that it works without it.

Reply:
It is Makefile, so it is used only at build time; if it did not crash during the build then it works.